README.md

```diff
@@ -352,6 +352,20 @@ const { winc, actualTokenAmount, equivalentWincTokenAmount } =
   });
 ```
 
+#### `getFiatEstimateForBytes({ byteCount, currency })`
+
+Get the current price from the Turbo Payment Service, denominated in the specified fiat currency, for uploading a specified number of bytes to Turbo.
+
+```typescript
+const turbo = TurboFactory.unauthenticated();
+const { amount } = await turbo.getFiatEstimateForBytes({
+  byteCount: 1024 * 1024 * 100,
+  currency: 'usd', // specify the currency for the price
+});
+
+console.log(amount); // Estimated usd price for 100 MiB
```

Comment:
Thoughts on including the formatted json output in the readme when we have these kinds of example lines? Otherwise, I'm not sure there's much of a point of it in the example. It's like saying "you can log it! in the same way you can log... anything..." which isn't very useful. As opposed to "and if you log it you'll see THIS" (which might be exactly what someone is looking for or curious about)

Reply:
f3420ba